pydantic/_internal/_core_utils.py

```diff
@@ -366,6 +366,13 @@ def handle_model_fields_schema(self, schema: core_schema.ModelFieldsSchema, f: W
         if 'extra_validator' in schema:
             schema['extra_validator'] = self.walk(schema['extra_validator'], f)
         replaced_fields: dict[str, core_schema.ModelField] = {}
+        replaced_computed_fields: list[core_schema.ComputedField] = []
+        for computed_field in schema.get('computed_fields', None) or ():
+            replaced_field = computed_field.copy()
+            replaced_field['return_schema'] = self.walk(computed_field['return_schema'], f)
+            replaced_computed_fields.append(replaced_field)
+        if replaced_computed_fields:
+            schema['computed_fields'] = replaced_computed_fields
```

Comment:
Is `BaseModel` the only class that takes the `@computed_field`s in consideration?

Reply:
Good point, added tests.